src/plugins/js-eval/__tests__/jsEvalPlugin-test.js

```diff
@@ -1,60 +1,108 @@
 const jsEval = require('../jsEvalPlugin');
 
 describe('jsEvalPlugin', () => {
-  it(`works`, () => {
-    jsEval({
+  it(`works`, async () => {
+    await jsEval({
       message: 'n> 2+2',
       respond: output => {
         expect(output).toEqual('(okay) 4');
       }
     });
+
+    await jsEval({
+      message: 'n> setTimeout(() => console.log(2), 1000); 1',
+      respond: output => {
+        expect(output).toEqual('(okay) 12\n');
+      }
+    });
   });
 
-  it(`errors when it should`, () => {
-    jsEval({
+  it(`errors when it should`, async () => {
+    await jsEval({
       message: 'n> 2++2',
       respond: output => {
-        expect(output).toEqual(`(error) ecmabot.js:1
+        expect(output).toEqual(`Error: ecmabot.js:1
```

Comment:
Can we remove this `ecmabot.js:1` stuff? Can be a separate PR.

Reply:
Hmm without [FILENAME](https://github.com/devsnek/docker-js-eval/blob/master/run.js#L15) 
```
Error: ecmabot.js:1\n1 ++ 1\n^\n\nReferenceError: Invalid left-hand side expression in postfix operation
```
changes to
```
Error: evalmachine.<anonymous>:1\n1 ++ 1\n^\n\nReferenceError: Invalid left-hand side expression in postfix operation
```

Reply:
done, only logging error name and message now https://github.com/brigand/jellobot/pull/22/commits/1f15b60680f72c888d2c5193715188cd025b294d#diff-8c048966e903e3957716694dd66d77f6

the `   ^` stuff to indicate where is the error is interesting, but less useful when the line-returns are removed